projects/reservoir/fv3/era_process.py

```diff
@@ -37,18 +62,34 @@ def main(args):
     setup_fregrid()
     for var in variables:
         check_time_steps_complete(path, var)
-        weekdays = ["mon", "tue", "wed", "thu", "fri", "sat", "sun"]
-        for i, day in enumerate(weekdays):
-            merge_into_file(path, var, i, day)
-            logger.info(f"{day} is merged")
-            regrid_to_cubed_sphere(path, var, day)
-            logger.info(f"{day} is regridded")
-            interpolated = interpolate_nans(var, day)
-            logger.info(f"{day} is interpolated")
-            masked = mask(interpolated)
-            logger.info(f"{day} is masked")
-            masked.to_netcdf(args.out_path + var + "_" + day + ".nc")
-            logger.info(f"{day} is done")
+        merge_into_weekly_file(path, var, args)
+        logger.info(f"{var} is merged into weekly file")
+        regrid_to_cubed_sphere(path, var)
+        logger.info(f"{var} is regridded")
+        interpolated = interpolate_nans(var)
+        interpolated.to_netcdf("interpolated.nc")
+        logger.info(f"{var} is interpolated")
+        masked = mask(interpolated)
+        masked.to_netcdf("temp.nc")
+        logger.info(f"{var} is masked")
+        if var in ["temp2m", "u_wind", "v_wind"]:
+            shifted = shift_variable(masked)
+        else:
+            shifted = masked
+        shifted.to_netcdf(
+            args.out_path + var + "_" + args.train_val_test + "_regridded.nc"
+        )
+        logger.info(f"{var} is done")
+    if "sst" in variables:
+        calculate_sst_tendency(args)
+        variables.append("sst_tendency")
+    merged = merge_variables(variables, args)
+    for var in variables:
+        var_id = var_id_mapping[var]
+        merged = merged.rename_vars({var_id: shift_rename_mapping[var_id]})
+    del merged.time.attrs["units"]
```

Comment:
Can move these lines modifying attrs into the `merge_variables` function.

Reply:
Actually, I removed the attrs stuff